apps/meteor/server/startup/cron.ts

```diff
@@ -12,14 +11,13 @@ import { videoConferencesCron } from '../cron/videoConferences';
 
 const logger = new Logger('SyncedCron');
 
-Meteor.defer(async () => {
+export const startCronJobs = async (): Promise<void> => {
 	await startCron();
```

Comment:
Maybe we could compile this into a `Promise.all()`?

Not really an issue, just a suggestion.